src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -35,24 +45,80 @@ const visibleColumns = [
 ];
 
 const JobLogsContainer = props => {
-  const { intl } = props;
+  const {
+    intl, stripes: { okapi },
+  } = props;
+
+  const calloutRef = useRef(null);
 
   const hasLoaded = true;
 
+  const handleDownloadError = () => {
+    if (!calloutRef.current) return;
+
+    calloutRef.current.sendCallout({
+      type: 'error',
+      message: <FormattedMessage id="ui-data-export.communicationProblem" />,
+    });
+  };
+
+  const getFileDownloadLink = async record => {
```

Comment:
According to the function name, the function is used to get link to download  the file. But it seems, it is used even to download file. Maybe, we should rename it. Or do I misunderstand the idea?